global.json

```diff
@@ -0,0 +1,10 @@
+{
+  "sdk": {
+    "allowPrerelease": false,
+    "rollForward": "minor",
+    "version": "3.1.100"
+  },
+  "msbuild-sdks": {
+    "Microsoft.Build.Traversal": "2.2.3"
+  }
+}
```

Comment:
Note : 
This is for appveyor to know should use this version of .net core to compile project.